From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Stephan Gerhold <stephan@gerhold.net>
Date: Tue, 28 Apr 2020 16:14:50 +0200
Subject: [PATCH 130/159] hack! ASoC: dt-bindings: Add Q6 Voice DAI binding

The Q6 Voice DAI driver exposes the Q6 Voice subsystem to ASoC.
At the moment it provides a single CS-Voice DAI with a hostless FE.
Eventually usage should be simplified using a codec2codec link.

[David]
 - split from "ASoC: qdsp6: Add Q6 Voice DAI driver (v2.1)"

Signed-off-by: Stephan Gerhold <stephan@gerhold.net>
(cherry picked from commit fe7841409608c17b02398ef8e592f5d6ba09b53e)
Link: https://github.com/msm8916-mainline/linux
Signed-off-by: Richard Acayan <mailingradian@gmail.com>
[Filippo Falezza: dropped the MAINTAINERS changes]
Signed-off-by: Filippo Falezza <filippo.falezza@outlook.com>
---
 include/dt-bindings/sound/qcom,q6voice.h | 8 ++++++++
 1 file changed, 8 insertions(+)
 create mode 100644 include/dt-bindings/sound/qcom,q6voice.h

diff --git a/include/dt-bindings/sound/qcom,q6voice.h b/include/dt-bindings/sound/qcom,q6voice.h
new file mode 100644
index 000000000000..3c6552bd2c56
--- /dev/null
+++ b/include/dt-bindings/sound/qcom,q6voice.h
@@ -0,0 +1,8 @@
+/* SPDX-License-Identifier: GPL-2.0 OR MIT */
+#ifndef __DT_BINDINGS_Q6_VOICE_H__
+#define __DT_BINDINGS_Q6_VOICE_H__
+
+#define CS_VOICE	0
+#define VOICEMMODE1	1
+
+#endif /* __DT_BINDINGS_Q6_VOICE_H__ */
